sktime/forecasting/structural.py

```diff
@@ -229,6 +232,7 @@ def __init__(
         optim_hessian=None,
         flags=None,
         low_memory=False,
+        approximate_diffuse_variance=None,
```

Comment:
same here, this param is not used..

Reply:
well but it looks to me like its always `None` and not exposed to user. The `initialize_default` method looks to me more like a private method actually, not sure why its not private. So we dont need it probably?

Reply:
If it should be private method then we won't need it. I'll undo this commit.